lib/LaTeXML/Core/Definition/Register.pm

```diff
@@ -73,6 +74,16 @@ sub invoke {
   LaTeXML::Core::Definition::stopProfiling($profiled, 'digest') if $profiled;
   return; }
 
+sub xxxxxxxxxxxequals {
+  my ($self, $other) = @_;
+  return (defined $other
+      && (ref $self) eq (ref $other)) && Equals($self->getParameters, $other->getParameters)
+###    && Equals($$self{replacement},  $$other{replacement})
+    && ($self->isRegister eq $other->isRegister)
+    && Equals($$self{getter}, $$other{getter})
+##    && Equals($$self{beforeDigest}, $$other{beforeDigest})
+##    && Equals($$self{afterDigest},  $$other{afterDigest})
+    ; }
 #===============================================================================
 1;
 
```

Comment:
Can we checkout this back to master (pre *my* pointless commit); it's a no-op.

Reply:
Confused by the phrasing "Check it out back to master". Do you mean remove the code block entirely, or do you mean "it's in master, let's rebase"?

Reply:
I must meant like ```git checkout ...\Register```: ie, no change to the file at all, since the only changes have no effect.

Reply:
Alright, I've removed the changes. Sadly "reverting" to the previous Register isn't simple this many commits later, I just papered over with a new commit.